vendor/digital-matter/oyster-codec.yaml

```diff
@@ -1,18 +1,36 @@
 uplinkDecoder:
   fileName: oyster.js
   examples:
-    - description: Position update
+    - description: Position update fix valid
```

Comment:
@jpmeijers as warnings is in return it needs an empty string in output ` "warnings":[] `

Reply:
Ahh ok. Tricky to figure this out. Was expecting to only change the js decoder, but realised I need to update the codec too, which I'm not certain about.

Reply:
yes, I'm was pushing the change and it makes me fetch and I see your change, thank you!